### PR TITLE
Fix docs for Runner class and file headers everywhere

### LINKED ISSUE #126: Improve class docs for Runner.php
**Issue Overview**

https://github.com/wearerequired/traduttore/blob/8e76cca/inc/Runner.php#L1-L19 includes references to the `Project` class.

### PR DESCRIPTION
**Pull request by swissspidy**
Fixes #126.

**Checklist:**
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code has proper inline documentation.

### CODE REVIEW

File: inc/Runner.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 use WP_Filesystem_Base;
 
 /**
- * Project class.
+ * Runner class.
```
Comment (ocean90) [inline inc/Runner.php]: I'd expect a comment starting with "Class used to…"

File: inc/Runner.php [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?php
 /**
- * Project class.
+ * Runner class.
```
Comment (ocean90) [inline inc/Runner.php]: According to the [docs standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#6-file-headers) there should be no period.

File: inc/Runner.php [hunk 3]
```diff
@@ -12,7 +12,11 @@
 use WP_Filesystem_Base;
 
 /**
- * Project class.
+ * Runner class.
```
Comment (ocean90) [inline inc/Runner.php]: This is just duplicating the file header, let's remove it? See also https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#2-classes.

File: inc/Runner.php [hunk 4]
```diff
@@ -12,7 +12,11 @@
 use WP_Filesystem_Base;
 
 /**
- * Project class.
+ * Project update runner.
```
Comment (ocean90) [inline inc/Runner.php]: I still have no clue what this should tell someone who wants to know what this class actually does but seems like you're not going to change it. 🤷‍♂️

Comment (swissspidy) [reply inc/Runner.php]: There's a longer explanation in the description below 🤔 

If that still is unclear for you then you're welcome to change this yourself. I can't read minds 🙂

Comment (ocean90) [reply inc/Runner.php]: Why does the "longer" description have to be below? 🤔 

"Project update runner." simply doesn't match what core's handbook is advising for an informative summary of a class.

Comment (swissspidy) [reply inc/Runner.php]: Never said we're following the core standards for docs 😅 One's gonna read the whole docblock, so this is more bikeshedding at this point than anything else.

But anyway, I can update it to something like `Update runner class to update a project's strings given a specific loader and an updater.` if that sounds better.

**Pull request merged**